internal/pkg/controllers/spod/bindata/spod.go

```diff
@@ -39,6 +40,7 @@ const (
 	SelinuxDropDirectory = "/etc/selinux.d"
 	SelinuxdSocketDir    = "/var/run/selinuxd"
 	SelinuxdSocketPath   = SelinuxdSocketDir + "/selinuxd.sock"
+	KmsgDropDirectory    = "/dev/kmsg"
```

Comment:
This var does not have to be public, right?

Reply:
right on, just updated that too, PTAL